docs/image-sizes.md

```diff
@@ -12,6 +12,16 @@ They have default dimensions, but you can override these either with code or in
 
 ![Media Settings](./assets/media-settings.png)
 
+### `content_width`
+
+If your theme defines a content width via the `$content_width` (or `$GLOBALS['content_width]`) global, this size will _override_ the size defined on the Settings > Media page. This will result in the Large size appearing to be the "wrong" size (using the size defined in the _global_ rather than the size in the admin) in the media modal.
+
+To make sure these sizes don't come into conflict, set the `$content_width` global to a large enough size that the admin setting won't override it.
```

Comment:
This seems to describe the opposite effect? The admin settings are what will be overridden.

Reply:
👍 Going to add clarity to the previous paragraph and just remove this sentence.